pages/linux/wg.md

```diff
@@ -0,0 +1,20 @@
+# wg
+
+> Manage the configuration of WireGuard interfaces.
+> More information: <https://www.wireguard.com/quickstart/>.
+
+- Check status of currently active interfaces:
+
+`sudo wg`
+
+- Print out a new private key:
+
+`wg genkey`
+
+- Print out a new public key:
+
+`echo {{private_key}} | wg pubkey`
+
+- Generate a public and private key into files:
+
+`wg genkey | tee {{privatekey}} | wg pubkey > {{publickey}}`
```

Comment:
If you added a common file extension to these file names it would make it very clear that they're supposed to be files.
```suggestion
`wg genkey | tee {{privatekey.txt}} | wg pubkey > {{publickey.txt}}`
```